src/routing_table/network_tests.rs

```diff
@@ -316,11 +316,13 @@ pub fn verify_network_invariant<'a, T, U>(nodes: U)
             }
             if prefix1.is_compatible(prefix2) {
                 panic!("Group prefixes should be disjoint, but these are not:\n\
-                    Group {:?}: {:?}\n\
-                    Group {:?}: {:?}",
+                    Group {:?}, according to node {:?}: {:?}\n\
+                    Group {:?}, according to node {:?}: {:?}",
```

Comment:
This is already my second implementation of this extension. I had nothing to test with when I first wrote this output.